src/include/hip/rpp_hip_roi_conversion.hpp

```diff
@@ -1,6 +1,9 @@
+#ifndef RPP_HIP_ROI_CONVERSION_H
+#define RPP_HIP_ROI_CONVERSION_H
+
 #include <hip/hip_runtime.h>
 
-extern "C" __global__ void roi_converison_ltrb_to_xywh(int *roiTensorPtrSrc)
+static __global__ void roi_converison_ltrb_to_xywh(int *roiTensorPtrSrc)
```

Comment:
why are these functions static?

Reply:
Static to limit scope to the current object file being compiled. Else gives a linker error.